README.md

```diff
@@ -266,10 +305,10 @@ The following instructions show you how to get data from an AWS S3 bucket
     $ cp api/v1/db/awsS3/pets-dao-example.js api/v1/db/awsS3/<resources>-dao.js
     ```
 
-4. Make sure to require the correct path for the new DAO file at path handlers files:
+4. Make sure to use the correct path for the new DAO file at path handlers files:
 
     ```js
-    const petsDao = require('../db/awsS3/<resources>-dao');
+    import petsDao from '../db/awsS3/<resources>-dao';
     ```
 
```

Comment:
Can we also specify what version of ECMAScript we are using in a badge of the very top of the README file?

Reply:
From our conversation yesterday, Babel doesn't have complete support for versions of ECMAScript so it might be better to have a badge for our Babel version. We could also reference this [table](https://kangax.github.io/compat-table/es6/). When we figure out the `core-js` version stuff, we may want to include something about that as well